langchain/src/vectorstores/closevector_web.ts

```diff
@@ -0,0 +1,276 @@
+import {
```

Comment:
Similar feedback to above

Reply:
I also wonder if it'd be possible to share more code between the two implementations

Reply:
also fixed this. :)